code/go/0chain.net/validatorcore/storage/models.go

```diff
@@ -61,6 +71,15 @@ func (r *DirMetaData) CalculateHash() string {
 	return encryption.Hash(strings.Join(childHashes, ":"))
 }
 
+func SortChildren(children []ObjectEntity) []ObjectEntity {
```

Comment:
Sorting is not required in validator.
Remove this function as well

Reply:
Removed